sdk/src/utils/font_thumbnail.rs

```diff
@@ -182,30 +161,33 @@ fn get_buffer_with_pt_size_fits_width(
         borrowed_buffer.set_size(width, height);
         borrowed_buffer.set_wrap(cosmic_text::Wrap::Glyph);
         borrowed_buffer.set_text(text, attrs, cosmic_text::Shaping::Advanced);
+        borrowed_buffer.shape_until_scroll(true);
         // Get the number of layout runs, we expect one if it fits on one line
         let count = borrowed_buffer.layout_runs().count();
         // If it is one, we have found the right size
         if count == 1 {
             let size = measure_text(text, attrs, &mut borrowed_buffer)?;
             // There instances where the measured width was 0, but maybe this is
             // caught now by counting the number of layout runs?
-            if size.w > 0.0 && size.w <= width && size.h <= height {
+            if size.w > 0.0 && size.w * 1.2 <= width {
```

Comment:
haha, `1.2` here, but `1.1` lower - maybe make them as constants so we can explain the rationale?

Reply:
I actually meant to remove that one. The `1.1` below is for a height buffer, but will move to a constant and do so there.